### PR TITLE
fix: issues with user menu

### PR DESCRIPTION
**Pull request by damianmuti**
* Updated styles and logic regarding ellipsis.
* Removed unused style specificity.
* Cleaned up styles.

### CODE REVIEW

**Comment by andresmoschini** [comment]
Hi @damianmuti,

image

There is a minor issue with _prettier_ format, you can fix it easily running `yarn prettier`.

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ class HeaderUserMenu extends React.Component {
                 ''
               )}
               <p>
-                <strong>{user.plan.remainingCredits}</strong>
+                <strong>{user.plan.remainingCredits}</strong>{' '}
```
Comment (andresmoschini) [inline src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: 👍

**Comment by damianmuti** [comment]
@andresmoschini forgot about running prittier; thanks for the reminder! 🙌

File: src/components/Header/HeaderUserMenu/HeaderUserMenu.js (hunk 1 above)
Comment (damianmuti) [reply src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: In the update of this PR, Prettier removed that space and put everything in the same line; works too, but wanted to tell you the differente, @andresmoschini

Comment (andresmoschini) [reply src/components/Header/HeaderUserMenu/HeaderUserMenu.js]: Well, if it works and it is _pretty_, it is fine for me. ;)

**Comment by andresmoschini** [comment]
🚀

**Comment by andresmoschini** [comment]
:shipit:

**Comment by andresmoschini** [comment]
Hi @damianmuti, I have rebased and merge in order to have this little fix already integrated:

image

I made a backup of your branch if you need something from there: https://github.com/FromDoppler/doppler-webapp/tree/DBR-100-user-menu-issues-BACKUP

**Pull request merged**